src/main/java/frc/robot/Constants.java

```diff
@@ -15,10 +15,22 @@
  * constants are needed, to reduce verbosity.
  */
 public final class Constants {
-    public static int DRIVE_MOTOR_1_ID = 1;
-    public static MotorType DRIVE_MOTOR_1_TYPE = MotorType.TALON_SRX;
-    public static int DRIVE_MOTOR_2_ID = 3;
-    public static MotorType DRIVE_MOTOR_2_TYPE = MotorType.TALON_SRX;
+    public static boolean DRIVE_DUAL_MOTORS = true;
+    public static int DRIVE_MOTOR_LEFT_LEADER_ID = 1;
+    public static MotorType DRIVE_MOTOR_LEFT_LEADER_TYPE = MotorType.TALON_SRX;
+    public static boolean DRIVE_MOTOR_LEFT_LEADER_INVERTED = false;
+
+    public static int DRIVE_MOTOR_LEFT_FOLLOWER_ID = 2;
+    public static MotorType DRIVE_MOTOR_LEFT_FOLLOWER_TYPE = MotorType.TALON_SRX;
+    public static boolean DRIVE_MOTOR_LEFT_FOLLOWER_INVERTED = false;
```

Comment:
I think the leader and follower will always be the same motor controller type, maybe we just need one. I think that might be true for the inverted flag, too.

Reply:
Actually, left and right will also always use the same motor controller types, so we can combine those as well.

Reply:
We will fix this in #2 since we are overhauling the entire constants system